<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询统计</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/back.jpg);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0" alignment="Qt::AlignTop">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="doctorPre">
     <property name="text">
      <string>指定医生处方情况</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="departmentNum">
     <property name="text">
      <string>各科室挂号人数</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="IncomeButton">
     <property name="text">
      <string>挂号费收入</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="departmentPre">
     <property name="text">
      <string>各科室处方金额</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>查询统计</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignLeft">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>选择查询日期：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="returnHome">
     <property name="text">
      <string>返回主界面</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="background_image.qrc"/>
 </resources>
 <connections/>
</ui>
